build/repo.targets

```diff
@@ -92,20 +92,22 @@
       <_LineupPackages Include="%(ArtifactInfo.PackageId)" Version="%(ArtifactInfo.Version)" Condition=" '%(ArtifactInfo.ArtifactType)' == 'NuGetPackage' " />
       <_LineupPackages Include="Microsoft.AspNetCore.All" Version="$(PackageVersion)" />
 
-      <_LineupSources Include="$(DependencyPackageDir)" Condition="'$(DependencyPackageDir)' != '' AND Exists('$(DependencyPackageDir)')" />
-      <_LineupSources Include="$(BuildDir)" />
-      <_LineupSources Include="$(IntermediateExternalPackageDir)" />
-      <_LineupSources Include="$(IntermediateMirrorPackageDir)" />
+      <_RestoreSources Include="$(DotNetRestoreSources)" Condition="'$(DotNetRestoreSources)' != ''" />
+      <_RestoreSources Include="$(DependencyPackageDir)" Condition="'$(DependencyPackageDir)' != '' AND Exists('$(DependencyPackageDir)')" />
+      <_RestoreSources Include="$(BuildDir)" />
+      <_RestoreSources Include="$(IntermediateExternalPackageDir)" />
+      <_RestoreSources Include="$(IntermediateMirrorPackageDir)" />
     </ItemGroup>
 
     <GeneratePackageVersionPropsFile
       Packages="@(_LineupPackages)"
-      OutputPath="$(GeneratedPackageVersionPropsPath)" />
+      OutputPath="$(GeneratedPackageVersionPropsPath)"
+      AdditionalImports="$(DotNetPackageVersionPropsPath)" />
```

Comment:
Not currently a parameter on this task. Does this require an update to buildtools?

Reply:
Can you commit the update to korebuild-lock.txt required to acquire https://github.com/aspnet/BuildTools/commit/166e381b9436b5c82434af3e53a5ffb63092e525?